### ISSUE TITLE
Router.js line 209 trim() function does not work on old browsers

### ISSUE THREAD

**Issue by gabsoftware** [brunotavares/Ext.ux.Router#4]
The following line causes old browsers to fail:

```
uri = (uri||"").toString().trim();
```

Fix: add the following code earlier in your code:

```
if(typeof String.prototype.trim !== 'function')
{
    String.prototype.trim = function()
    {
        return this.replace(/^\s+|\s+$/g, '');
    }
}
```

**Comment by brunotavares** [comment]
Thanks for the report. I'm currently working on a major change that will make the router engine way better. You can check a preview on the "pre" branch 

https://github.com/brunotavares/Ext.ux.Router/tree/pre

**Comment by gabsoftware** [comment]
Hi,

Thank you for your work, it's awesome!

**Comment by brunotavares** [comment]
Thanks for the feedback. It should be working properly now with the new version. I already merged "pre" into "master" with the new v2.0 tag.
